superset/assets/src/dashboard/components/Header.jsx

```diff
@@ -1,12 +1,18 @@
+/* eslint-env browser */
 import React from 'react';
 import PropTypes from 'prop-types';
-import { ButtonGroup, ButtonToolbar } from 'react-bootstrap';
+import {
+  DropdownButton,
+  MenuItem,
+  ButtonGroup,
+  ButtonToolbar,
+} from 'react-bootstrap';
 
 import Controls from './Controls';
 import EditableTitle from '../../components/EditableTitle';
 import Button from '../../components/Button';
 import FaveStar from '../../components/FaveStar';
-import SaveModal from './SaveModal';
+import SaveModal, { SAVE_TYPE_OVERWRITE } from './SaveModal';
```

Comment:
I think `SAVE_TYPE_OVERWRITE ` should be moved into the dashboard constants file, this defeats the purpose of having that file and centralizes all constants.